.jenkins/actions/lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -e -x
+env_name=venv-${BUILD_NUMBER:-0}
+python3 -m venv ${env_name}
+. ${env_name}/bin/activate
+pip install -r fv3core/requirements/requirements_lint.txt -c constraints.txt
```

Comment:
We really should hard-code `pip install pre-commit` here since that's clearly the one command/package we're using for this action.